contracts/modules/SSVClusters.sol

```diff
@@ -850,21 +699,17 @@ contract SSVClusters is ISSVClusters {
 
         if (!liq) return;
 
-        _performLiquidation(
-            clusterOwner,
-            clusterId,
-            operatorIds,
-            cluster,
-            version
-        );
+        _executeLiquidation(clusterOwner, clusterOwner, clusterId, operatorIds, cluster, version, false);
```

Comment:
@venimir-ssv here, the 2nd parameter should be msg.sender, as the reward will always go to the caller. I think this was not introduced by you, but please change it 🙏🏻